calico-enterprise/operations/monitor/prometheus/support.mdx

```diff
@@ -10,10 +10,22 @@ description: Prometheus support in Calico Enterprise.
 
 ## Install options
 
-- Use Prometheus operator managed by Tigera operator
+### Use Prometheus operator managed by Tigera operator
 
-  You install the {{prodname}} Prometheus operator and CRDs during {{prodname}} installation. {{prodname}} metrics and alerts are available in Manager UI. You configure alerts through Prometheus AlertManager.
+You install the {{prodname}} Prometheus operator and CRDs during {{prodname}} installation. {{prodname}} metrics and alerts are available in Manager UI. You configure alerts through Prometheus AlertManager.
 
-  If you want to specify your own Prometheus operator during installation for management by the Tigera operator, the require operator version must be **v0.40.0 or higher**. Because {{prodname}} creates AlertManager and Prometheus CRs in the `tigera-prometheus` namespace, all you need to do is verify that your Prometheus operator is configured to manage Prometheus and AlertManager instances in the `tigera-prometheus` namespace.
+If you want to specify your own Prometheus operator during installation for management by the Tigera operator, the require operator version must be **v0.40.0 or higher**. Because {{prodname}} creates AlertManager and Prometheus CRs in the `tigera-prometheus` namespace, all you need to do is verify that your Prometheus operator is configured to manage Prometheus and AlertManager instances in the `tigera-prometheus` namespace.
```

Comment:
```suggestion
If you want to specify your own Prometheus operator during installation for management by the Tigera operator, the required operator version must be **v0.40.0 or higher**. Because {{prodname}} creates AlertManager and Prometheus CRs in the `tigera-prometheus` namespace, all you need to do is verify that your Prometheus operator is configured to manage Prometheus and AlertManager instances in the `tigera-prometheus` namespace.
```